Applications/MmrMobile/Contracts/AlertEventCausality.cs

```diff
@@ -0,0 +1,89 @@
+﻿namespace HomagConnect.MmrMobile.Contracts
+{
+    /// <summary>
+    /// What process/part of the production is to blame for the Event
+    /// </summary>
+    public enum AlertEventCausality
+    {
+        /// <summary>
+        /// Machine is the causality />
+        /// </summary>
+        CausalityMachine,
```

Comment:
Is the prefix "Causality" needed?

Reply:
This are the enum-Values from woodscout. I can remove the prefix